.github/workflows/assign-pr-to-creator.yml

```
@@ -0,0 +1,24 @@
+name: Assign PR to creator
+
+on:
+  pull_request: 
+    types:
+      - reopened
+      - opened
+
+permissions:
+  pull-requests: write
+
+jobs:
+  assign-pr-to-creator:
+    name: Assign PR to creator
+    runs-on: ubuntu-latest
+    steps:
+      - name: Assign PR to creator
+        if: ${{ github.event_name == 'pull_request' }}
+        run: |
+          gh pr edit $PR --add-assignee $AUTHOR
+        env:
+          AUTHOR: ${{ github.event.pull_request.user.login }}
+          PR: ${{ github.event.pull_request.html_url }}
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

```suggestion
          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}

```

Git likes an empty line at the end.